<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Данные клиента</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="client_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Иван Иванов</string>
    </property>
   </widget>
   <widget class="QLabel" name="client_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NONAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="client_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить пароль</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
